### DISCUSSION TITLE
Missing Starstruck badge

### DISCUSSION THREAD

**Discussion by ahmadalwareh** [community/community#43941] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I've gained 16 stars on this [repository](https://github.com/ahmadalwareh/Python_Interview_Questions) but I did not get a Starstruck badge under the [Achievements](https://github.com/ahmadalwareh?tab=achievements)  

Am I missing something or this criterion is not met?

**Comment by melvincwng** [comment, marked as answer]
@ahmadalwareh It takes a while for the badge to arrive.

![image](https://user-images.githubusercontent.com/77479885/211803460-f48875b6-00c3-4036-b840-8d2f7a132794.png)

This is your latest 'Achievements' section! I can see the 'Starstruck' badge over there :smile:

Congratulations :star:

**Reply by ahmadalwareh** [reply]
Thanks @melvincwng !

But this was given regarding this [repo](https://github.com/ahmadalwareh/DevOps)
But for this [repo](https://github.com/ahmadalwareh/Python_Interview_Questions) I did not receive any :/

**Reply by melvincwng** [reply]
@ahmadalwareh, Ohhh so that was your question. Ok got it now!

For the 'Starstruck' achievement, the first repository that achieved 16 stars would appear on the profile. The other repository wouldn't appear.

You only receive it once at 16 stars, the next one at 128, the third one at 512, and the final gold colored one at 4096 :star:

Hope this clarifies & answers your question, have a good day 😄

**Reply by melvincwng** [reply]
Basically in summary, you already achieved the badge from your 'DevOps' repository, hence the other 'Python' repository (even though it may be 16+ stars), will not garner you another badge.

You can only display **one** Starstruck badge at any time.

**Reply by ahmadalwareh** [reply]
What about #YOLO ?
I've done multiple merges without a review and did not receive it also.
@melvincwng

**Reply by melvincwng** [reply]
@ahmadalwareh, check out these two links for achievements references:

- https://github.com/community/community/discussions/18467#discussioncomment-2934894
- https://github.com/Schweinepriester/github-profile-achievements

Hope this helps & answers your question ^^

**Comment by Not-Safe-Any-System** [comment]
Hi GitHub Support,

My GitHub username is: Not-Safe-Any-System

I have starred over 100 repositories, and my "Show Achievements on my profile" setting is enabled. However, I still haven’t received the “Starstruck” achievement badge.

I’ve tried starring new repositories and refreshing my profile, but the badge still doesn’t appear. Please help me resolve this issue.

Thanks in advance!
